lua/trevj.lua

```diff
@@ -1,9 +1,7 @@
 local M = {}
 
 local ts = vim.treesitter
-local get_node_text = vim.treesitter.get_node_text or vim.treesitter.query.get_node_text
-local ts_utils = require("nvim-treesitter.ts_utils")
-local parsers = require("nvim-treesitter.parsers")
+local get_node_text = ts.get_node_text or ts.query.get_node_text
```

Comment:
maybe this would be removed now, not sure exactly but it seems `ts.get_node_text` was added in 0.9?

Reply:
I find it hard to know which version a certain function was added. For example when was `ts.get_node` added? If that was 0.9 or later we're now anyway requiring that